crates/driver/src/domain/eth/mod.rs

```diff
@@ -197,17 +204,17 @@ impl From<ContractAddress> for Address {
 ///
 /// https://eips.ethereum.org/EIPS/eip-20
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct TokenAddress(pub H160);
+pub struct TokenAddress(pub ContractAddress);
```

Comment:
This is interesting - I would have kept `TokenAddress(pub H160)` and have a conversion function `TokenAddress -> ContractAddress`.

Not suggesting to change this, I'm more just curious about the rationale of wrapping multiple "marker" types.

Reply:
Makes sense.